components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -171,7 +171,9 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
     const isPrebuilt = WithPrebuild.is(this.state.workspace?.context);
     let phase = StartPhase.Preparing;
     let title = undefined;
-    let statusMessage = <p className="text-base text-gray-400">Preparing workspace …</p>;
+    let statusMessage = !error
+      ? <p className="text-base text-gray-400">Preparing workspace …</p>
+      : <p className="text-base text-red-500">{error.message}</p>;
```

Comment:
**question**: Can we safely limit the width here (`max-width`) with the [futuristic](https://github.com/gitpod-io/gitpod/pull/3560#discussion_r602123684) `w-96` and center align all text?